components/modelregistry/modelregistry.go

```diff
@@ -0,0 +1,84 @@
+// Package modelregistry provides utility functions to config ModelRegistry, an ML Model metadata repository service
+package modelregistry
+
+import (
+	"context"
+	"path/filepath"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/components"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+)
+
+var (
+	ComponentName = "model-registry-operator"
```

Comment:
is there a reason for the component name to be different from the label used in the DSC? "modelregistry" vs "model-registry-operator". While they can be different, it usually causes less confusion if they are the same.

Reply:
Good question @etirelli . I was going by what DSP has done. It looks like there are multiple places where components are referenced, and this one happens to refer to the github project, which is the `model-registry-operator` project. The config property in the DSC CR is called modelregistry from `apis/datasciencecluster/v1/datasciencecluster_types.go`.